CONTRIBUTING.md

```diff
@@ -93,15 +109,28 @@ To leave the virtual environment, simply use:
 deactivate
 ```
 
-### Running the unit tests
+### Running a complete local build
 
 Make sure you have created a Python virtual environment for the Phantom project as described above.
 
-Then, activate the virtual environment and install the dependencies before running the tests.
+Then, activate the virtual environment and install the dependencies before running the build.
 
 ```bash
 pip install -e .[dev]
-pytest
+make
+```
+
+By default, `make` will run all unit tests, Phantom configuration / lint validators, style checks, and create a local tarball of the app.
+
+Each of these commands can also be run individually if desired:
+
+```bash
```

Comment:
I am a little sad you removed the `ssh` and the `open-web` commands, but I can make custom aliases for those if you don't think they belong. I was using them though.

Reply:
I didn't mean to leave them out! I can add them back, sorry about this.

Reply:
back in now!